lib/storage/http.js

```diff
@@ -22,15 +22,22 @@ const DEFAULT_CONTENT_TYPE = 'application/octet-stream';
 
 async function download(source, file) {
     try {
-        const contentType = mime.contentType(source.name)  || DEFAULT_CONTENT_TYPE;
+        const contentType = mime.contentType(source.name) || DEFAULT_CONTENT_TYPE;
         console.log(`downloading source: ${AssetComputeLogUtils.redactUrl(source.url)} into ${file}`);
+
+        const headers = {
+            'content-type': contentType
+        };
+        if(source.headers){
```

Comment:
I don't think we need the headers check here. You can just change line 28 to be:
```
const headers = Object.assign({
            'content-type': contentType
        }, source.headers)
```
as it will just ignore it if `source.headers` is undefined:
<img width="635" alt="Screen Shot 2020-09-14 at 12 50 34 PM" src="https://user-images.githubusercontent.com/20048485/93131500-069b1280-f689-11ea-934c-9be77d6cbe70.png">

Reply:
I'm interested in the `console.log` in the `if` statement. We won't log Auth headers, so at least this indicates something custom/set through params came in. It will be useful to have that information if things fail.

Reply:
We log params.source already for nui requests. Just logging the final headers would be fine.

(But see above, this all might be moot).